tests/samples/input/ocaml.ml

```diff
@@ -495,3 +495,17 @@ let add_as_fun_multiline = fun x ->
   x
 
 let add_as_fun_one_line = fun x -> x
+
+let sum_of_int n =
```

Comment:
@aspiwack I'll approve this, but some multi-line elements are turned into single-line here, so you might want to take a look too.

Reply:
Indeed, there is definitely something fishy here. Even with the most generous interpretation of what a node is, this should not become a single line. Can someone open an issue to track that let-ins seem to attract things upward like this?

(in this particular case, I expect `sum_of_int` to be formatted as written)

Reply:
I can open an issue about it.

Reply:
:arrow_right: #134 .

Reply:
This can be merged, then.